### PR TITLE
Translation and full misspelling correction

### PR DESCRIPTION
**Pull request by NotDead**
Added some more ruRU translations
**please check my "Compatibility" modification!**

### CODE REVIEW

File: Carbonite/NxOptions.lua [hunk 1]
```diff
@@ -275,17 +275,17 @@ local function mapConfig ()
 								Nx.db.profile.Map.MaxOverride = not Nx.db.profile.Map.MaxOverride
 							end,
 						},
-						compatability = {
+						Compatibility = {
```
Comment (IrcDirk) [inline Carbonite/NxOptions.lua]: Dont know if this should be capital "C"... did u tested your changes?

**Comment by IrcDirk** [comment]
Added comment to one line. U did tested all those changes?

**Comment by Rythal** [comment]
It's an internal table header thing, not used anywhere so the spelling and capitalization doesn't matter

**Comment by IrcDirk** [comment]
Ok, good to know.

**Pull request merged**